dockers/docker-sonic-restapi/supervisord.conf

```diff
@@ -3,10 +3,26 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
-[program:restapi]
-command=/usr/sbin/go-server-server -loglevel trace
+[program:start.sh]
+command=/usr/bin/start.sh
 priority=1
 autostart=true
+autorestart=true
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:restapi]
+command=/usr/sbin/go-server-server -enablehttps=true -clientcert=/etc/sonic/certificates/restapiclient.crt -servercert=/etc/sonic/certificates/restapiserver.crt -serverkey=/etc/sonic/certificates/restapiserver.key -loglevel=trace
+priority=1
+autostart=false
 autorestart=false
-stdout_logfile=/tmp/rest-api.out.log
-stderr_logfile=/tmp/rest-api.err.log
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:rsyslogd]
+command=/usr/sbin/rsyslogd -n
+priority=2
+autostart=false
```

Comment:
autostart should be true.

Reply:
This is being started from the start script.

---
In reply to: [404614583](https://github.com/Azure/sonic-buildimage/pull/4383#discussion_r404614583) [](ancestors = 404614583)

Reply:
This is correct.